components/StatsTable/StatsTable.tsx

```diff
@@ -0,0 +1,42 @@
+import React from "react"
+import styles from "./StatsTable.module.scss"
+
+// where should this be exported from?
+interface IRow {
+  label: string
+  value: any
+}
+export type { IRow }
+
+interface IStatsTableProps {
+  stats: IRow[]
+  caption: string
+}
+
+function Row({ label, value }: IRow) {
+  return (
+    <tr>
+      <th>{label}</th>
+      <td>{value}</td>
+    </tr>
+  )
+}
+
+export default function StatsTable({ stats, caption }: IStatsTableProps) {
+  console.log(`
+    caption: ${caption}
+    stats: ${stats}
+  `)
+  return (
+    <div className={styles.wrapper}>
+      <table className={styles.table}>
+        <caption className="visually-hidden">{caption}</caption>
+        <tbody>
+          {stats?.map(({ label, value }) => (
+            <Row key={label} label={label} value={value} />
+          )) || null}
```

Comment:
null is actually redundant, I think.  if stats is undefined, then expression will return undefined and React will render empty content, same as null